packages/beacon-node/src/chain/opPools/syncCommitteeMessagePool.ts

```diff
@@ -66,6 +73,11 @@ export class SyncCommitteeMessagePool {
       throw new OpPoolError({code: OpPoolErrorCode.SLOT_TOO_LOW, slot, lowestPermissibleSlot});
     }
 
+    // validator gets SyncCommitteeContribution at 2/3 of slot, it's no use to preaggregate later than that time
+    if (this.clock.secFromSlot(slot) > this.cutOffSecFromSlot) {
+      throw new OpPoolError({code: OpPoolErrorCode.LATE_MESSAGE, slot});
```

Comment:
How loud would be this error?

Reply:
we throw error here and gossipHandler would log error there.
since this is mainly for debugging purpose, not for end user so I changed log level to debug.

Reply:
Wh not return an insert outcome? Then the caller can decide to maybe log a debug if result is not add